packages/examples/package.json

```diff
@@ -10,25 +11,23 @@
   "files": [
     "examples/"
   ],
+  "workspaces": [
+    "examples/*"
+  ],
   "scripts": {
     "setup": "yarn install && yarn allow-scripts",
-    "test": "echo 'N/A'",
-    "test:ci": "yarn test",
     "lint:changelog": "yarn auto-changelog validate",
     "lint:eslint": "eslint . --cache --ext js,ts",
     "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' --ignore-path ../../.gitignore",
     "lint": "yarn lint:eslint && yarn lint:misc --check",
     "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
-    "build": "node ./scripts/buildExamples.js",
-    "build:pre-tsc": "echo 'N/A'",
+    "build": "yarn workspaces foreach -pi run build",
```

Comment:
What do you think about using the full option names here instead of `-pi`?

Reply:
In my opinion, changing the commands to use `--parallel`, `--interlaced`, `--topological` etc. just adds clutter, and potentially makes it more difficult to read.

Reply:
I have to disagree on this point, because I immediately know or can at least guess what `--parallel` or `--topological` means for a given command, but I have no idea what `-pi` means. The former is to some degree self-documenting, while the latter will force the reader to consult external docs for any level of understanding. The shorthand commands are also more difficult to search for if I do decide to look them up in the docs.

Reply:
Alright, changed it to the full names.